src/content/clubLife.yaml

```diff
@@ -1,72 +1,55 @@
-- Title: Mentorship
+- Title: Beyond the boundaries
   Content: !markdown |
-    Here at FOSS@Amrita, seniors guide juniors on their way in the club. It is a learning and development
-    partnership between someone with vast experience and someone who wants to learn. Mentoring others is one
-    of the most important parts of being in a community. This is an experience and a skill which cannot be
-    gained in a classroom. In FOSS@Amrita when you get enough experience, you will be assigned mentees -
-    people whom you have to train. You will be responsible for tracking their progress, ensuring that they
-    are doing what is best suited for them.
+    Being a part of bi0s opens up the door to a multitude of oppurtunities. The members of bi0s have been selected
```

Comment:
Being a part of bi0s opens up the door to a multitude of opportunities. Members of bi0s have been selected

Reply:
Okay sure!